### PR TITLE
Fix breadcrumbs and MVVM TOC

### PR DESCRIPTION
**Pull request by michael-hawker**
Attempt to aid with fixing #54.

FYI @Sergio0694 as I've updated the layout for the MVVM TOC to hopefully make things a bit more discoverable. We'll see how it works out.

I did check that we have auto-generate of Breadcrumbs setting on, but think it may be getting tripped up by our structure or because of our multiple TOC files.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@michael-hawker : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@michael-hawker : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.
 Json File | This PR contains json file(s).

**Comment by michael-hawker** [comment]
Breadcrumbs do appear a bit better now:

image

image

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.
 Json File | This PR contains json file(s).
 Repository root file |  This pull request contains a file submitted to the root of the repository.

**Review by brminnick — APPROVED** [review]

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.
 Json File | This PR contains json file(s).
 Repository root file |  This pull request contains a file submitted to the root of the repository.

File: docs/index.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ### YamlMime:Landing
 
-title: .NET Community Toolkits documentation
+title: .Community Toolkits Documentation
```
Comment (michael-hawker) [inline docs/index.yml:3]:
```suggestion
title: Community Toolkits Documentation
```

Extra dot

**Comment by PRMerger19** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.
 Json File | This PR contains json file(s).
 Repository root file |  This pull request contains a file submitted to the root of the repository.

**Pull request merged**